src/data/screenshots-archive-2-6.json

```diff
@@ -169,7 +169,7 @@
             },
             {
                 "title": "Check certificates for validity before a trip",
-                "anchor": "ios_check_validity_certificate",
+                "anchor": "android_check_validity_certificate",
```

Comment:
Arguably this is not much better because the screenshots below are actually for iOS; however this is the Android section.
Maybe this section should be removed completely for Android then, in case no screenshots exist for it?